wiki/Rules/es.md

```diff
@@ -12,8 +12,9 @@ Estas reglas son la base sobre la cual mantenemos la comunidad de osu! como un l
 2. **Juega limpio.** Usar herramientas de terceros de cualquier tipo para obtener cualquier forma de ventaja no está bien. Esto incluye cosas como macros, programas de asistencia de puntería, modificación de escalas de tiempo, y demases. Si algún programa te está ayudando a hacer algo que deberías hacer tú mismo en el juego, no está bien.
```

Comment:
- Check if the article is fully up-to-date with the English version
   - If yes, delete the `outdated` mark at the beginning
- Link at line 22 to `Reporting Bad Behaviour` is missing
- `Reglas de Envio de Beatmaps` header is outdated
- Link at line 68 is missing (the whole header might be outdated?)